src/zzz_od/backend/mcp/config_app.py

```
@@ -0,0 +1,231 @@
+            return {'ok': True, 'app_id': app_id, 'list_field': list_field, 'id': new_id}
+        except Exception as e:  # noqa: BLE001
+            return {'ok': False, 'error': str(e)}
+    return add_config_item
+
+
+def make_delete_config_item(backend: ZzzBackendContext) -> Callable:
+    """构造 ``delete_config_item`` tool。"""
+    async def delete_config_item(
+        app_id: Annotated[str, Field(description="app 配置 id,如 charge_plan")],
+        list_field: Annotated[str, Field(description="列表字段名")],
+        id: Annotated[str, Field(description="项标识:charge_plan/notorious_hunt 用 plan_id;standalone_app/_group 用 app_id")],
```

Ruff (0.15.21)

[error] Function argument `id` is shadowing a Python builtin

(A002)